src/models/Accommodation.js

```diff
@@ -74,15 +78,15 @@ const AccommodationDefinition = (sequelize, DataTypes) => {
   // eslint-disable-next-line func-names
   Accommodation.findWhereStartedBy = function (startedBy) {
```

Comment:
Woof! Use ES6 arrow functions.

Reply:
Woof! I'm relying on  the "this" keyword in the function. It isn't possible to get it to work in an arrow function.